pkg/endpoint/endpoint.go

```diff
@@ -109,15 +109,6 @@ var _ notifications.RegenNotificationInfo = &Endpoint{}
 // Endpoint represents a container or similar which can be individually
 // addresses on L3 with its own IP addresses. This structured is managed by the
 // endpoint manager in pkg/endpointmanager.
-//
-//
-// WARNING - STABLE API
-// This structure is written as JSON to StateDir/{ID}/lxc_config.h to allow to
-// restore endpoints when the agent is being restarted. The restore operation
-// will read the file and re-create all endpoints with all fields which are not
-// marked as private to JSON marshal. Do NOT modify this structure in ways which
-// is not JSON forward compatible.
-//
```

Comment:
It may be good to leave a note here saying that the serializable bit is now in `restoredEndpoint`. This is mostly for backports (which I'm hoping won't apply cleanly once we merge in other changes to Endpoint) so they know where to look.

Reply:
Done!